lint-all.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
+
 set -e
 
 echo "Linting go sources"
-cd go && ./lint.sh && cd ..
+cd go && . ./lint.sh && cd ..
```

Comment:
This confuses me. Do we have to source the script to run it? Maybe we should rewrite the script?